atomic_reactor/config.py

```diff
@@ -483,7 +483,7 @@ def hide_files(self):
     @property
     def skip_koji_check_for_base_image(self):
         return self._get_value(ReactorConfigKeys.SKIP_KOJI_CHECK_FOR_BASE_IMAGE_KEY,
-                               fallback=False)
+                               fallback=True)
```

Comment:
Shouldn't be this rather config change?

Reply:
it could be, but since RH is mainly using this, I'd say having default is fine as well